invokeai/app/api_app.py

```diff
@@ -77,7 +77,12 @@
 
 # Create the app
 # TODO: create this all in a method so configuration/etc. can be passed in?
-app = FastAPI(title="Invoke AI", docs_url=None, redoc_url=None, separate_input_output_schemas=False)
+app = FastAPI(
+    title="Invoke AI",
+    docs_url=None,
+    redoc_url=None,
+    separate_input_output_schemas=False,
+)
```

Comment:
Why are all these whitespace changes in here? Aren't these the _same set_ of changes I _backed out_ of #5262 to get it to pass the format checks then? Why did ruff format add them back this time? :dizzy_face: